src/reporter/errors/templates.js

```diff
@@ -351,3 +307,81 @@ export default {
         ${getMsgPrefix(err, CATEGORY.pageLoadError)}${err.message}
     `)
 };
+
+const UNESCAPED_ERROR_PROPERTIES = ['actual', 'expected', 'dialog', 'action', 'type', 'userAgent',
+    'pageUrl', 'isArrays', 'isObjects', 'isStrings', 'diffType'];
+
+
+function escapeError (errorType, error) {
```

Comment:
I don't like this approach. The good thing about templates was it' declarative nature. Now we have some black magic going on under the hood. From quick observation I don't get which properties should be escaped? BTW, do we really need to escape a lot properties? I'm still for the inline escaping in the template code.